movement/analysis/kinematics.py

```diff
@@ -33,22 +49,34 @@ def compute_displacement(data: xr.DataArray) -> xr.DataArray:
 
 
 def compute_velocity(data: xr.DataArray) -> xr.DataArray:
-    """Compute the velocity in cartesian ``(x,y)`` coordinates.
+    """Compute velocity array in cartesian coordinates.
+
+    The velocity array is the first time derivative of the position
```

Comment:
Oh good point. I was thinking about this sentence as a general statement, "time" referring to the physical quantity, rather than the specific name of the array dimension. But it is true that we have changed the sentence to say "array"...

I am inclined to leave it without formatting, in line with the kinematic variable just before it in the same sentence (velocity, acceleration). I changed it to "time-derivative" to be a bit more explicit that it the general term we are talking about.